includes/orders/classes/class-order.php

```diff
@@ -384,8 +384,9 @@ public function get_adjustments() {
 	public function get_discounts() {
 		$discounts = array();
 
+		// Ensure adjustments exist.
 		if ( empty( $this->adjustments ) ) {
```

Comment:
I think the check for this should change to check for `null`, as the fees method does--if the adjustments have not been retrieved yet, that's what it will be, and this will set it, but if they have been retrieved, but there are no adjustments, `$this->adjustments` could still be empty. It _may_ not make a difference (this is working for sure), but I think it would be good to be consistent.

Reply:
Thanks.  I was going back and forth on this, but I couldn't decide which one was "right".